src/Widgets/WindowSwitcher.vala

```diff
@@ -522,22 +575,25 @@ namespace Gala
 
 		void dim_windows ()
 		{
-			var window_opacity = (int) Math.floor (AppearanceSettings.get_default ().alt_tab_window_opacity * 255);
-
 			foreach (var actor in window_clones.get_children ()) {
 				unowned SafeWindowClone clone = (SafeWindowClone) actor;
 
 				actor.save_easing_state ();
 				actor.set_easing_duration (250);
-				actor.set_easing_mode (AnimationMode.EASE_OUT_QUAD);
+				actor.set_easing_mode (AnimationMode.EASE_IN_OUT_QUART);
 
 				if (clone.window == current_window.window) {
 					window_clones.set_child_above_sibling (actor, null);
+					actor.remove_effect_by_name ("brightness");
 					actor.z_position = 0;
-					actor.opacity = 255;
 				} else {
-					actor.z_position = -200;
-					actor.opacity = window_opacity;
+					if (actor.get_effect ("brightness") == null) {
+						var brightness_effect = new BrightnessContrastEffect ();
```

Comment:
I've tried that before, this cannot be done as Clutter throws a warning that the same effect cannot be applied to another actor.